gap/TGQuotientSequences.gd

```diff
@@ -25,6 +25,127 @@
 #! @Arguments sequence
 DeclareGlobalFunction( "IsTGQuotientSequence" );
 
+
+#! @Section Constructing Sequences of Triangle-Group Quotients
+#!
+#!   Sequences of quotient groups can efficiently be identified through the construction of adjacency
+#!   matrices that capture the normal subgroup relation between pairwise distinct translation groups 
+#!   of corresponding quotients. 
+#!
+#!   The adjacency matrix is implemented as an object of category `TGQuotientSequencesAdjacencyMatrix` with the following
+#!   components:
+#!   - <Ref Oper='Signature' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     signature of the underlying triangle group
+#!   - <Ref Oper='BoundByGenus' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     upper bound of the triangle group quotient genus
+#!   - <Ref Oper='GetListTGQuotients' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     list of quotients with genus smaller than upper bound of the triangle group quotient genus
+#!   - <Ref Oper='MirrorSymmetries' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     binary list, 1 if quotients have mirror symmetries 0 otherwise
+#!   - <Ref Oper='IsSparse' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     boolean, if true the adjacency matrix is sparsely represented
+#!   - <Ref Oper='AdjacencyMatrix' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     adjacency matrix, which includes all quotients with and without mirror symmetries
+#!   - <Ref Oper='NearestNeighborAdjacencyMatrix' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     adjacency matrix of normal subgroup relation between consecutive translation groups of corresponding quotients
+#!   - <Ref Oper='LongestSequence' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     list of longest sequence of quotient groups 
+#!   and is printed in the form
+#!   @BeginLog
+#!   TGQuotientSequencesAdjacencyMatrix( 
+#!     [r, q, p], 
+#!     boundByGenus = int,
+#!     lstTGQuotients = [[genus, number], ... ],
+#!     lstMirrorSymmetries = [ 0 / 1 , ... ],
+#!     sparse = bool,
+#!     adjMatrix = [[ ... ], ... ]
+#!   )
+#!   @EndLog
+  
+DeclareCategory( "IsTGQuotientSequencesAdjacencyMatrixObj", IsObject );
+
+#! @Description
+#!   Construct the adjacency matrix for the triangle group <A>tg</A> ($\Delta^+$), given as ProperTriangleGroup object (see
+#!   2.1), specified by the quotient relators given as elements of <A>tg</A>.
```

Comment:
References (here "see 2.1") should not be hard coded, please use the corresponding label.
```suggestion
#!   <Ref Sect="Section_TriangleGroups"/>), specified by the quotient relators given as elements of <A>tg</A>.
```